src/Sylius/Behat/Context/Domain/OrderContext.php

```diff
@@ -137,4 +137,20 @@ public function adjustmentShouldNotExistInTheRegistry()
         $adjustments = $this->adjustmentRepository->findBy(['id' => $adjustments]);
         expect($adjustments)->toBe([]);
     }
+
+    /**
+     * @Given I am logged in as an administrator
+     */
+    public function iAmLoggedInAsAnAdministrator()
+    {
+        // Not applicable in the domain scope
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully deleted
+     */
+    public function iShouldBeNotifiedOfSuccess()
```

Comment:
I dont think so, because this is an implementation of step on domain level, so any step with notification could be implemented in this method in my opinion.